### PR TITLE
Support masking for tf2 and tf.keras

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

File: keras2onnx/_parse_tf.py [hunk 1]
```diff
@@ -269,14 +269,28 @@ def build_layer_output_from_model(model, output_dict, input_names, output_names)
         return graph
 
 
+def _get_layer_endpoints(layer_endpoints, layer_info_end_points):
```
Comment (wenbingl) [inline keras2onnx/_parse_tf.py:272]: Can you leave some comments about this function?
